app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -76,6 +76,7 @@ public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article
                 'name' => $category->name,
                 'description' => $category->description,
             ]),
+            'parentCategory' => $category->parentCategory,
```

Comment:
It may be best if this here and in other places was a part of the `KnowledgeBaseCategoryData` used in the above `category` item rather than stand alone.